src/cli/log.rs

```diff
@@ -18,26 +18,6 @@ use opentelemetry_sdk::trace::Tracer;
 
 use crate::{currentprocess::Process, utils::notify::NotificationLevel};
 
-macro_rules! debug {
-    ( $ ( $ arg : tt ) * ) => ( ::tracing::trace ! ( $ ( $ arg ) * )  )
-}
-
-macro_rules! verbose {
-    ( $ ( $ arg : tt ) * ) => ( ::tracing::debug ! ( $ ( $ arg ) * )  )
-}
```

Comment:
The nomenclature trap is here. Every time `verbose!()` is used, you should replace it with `tracing::debug!()` instead; every time `debug!()` is used, you should **replace it with `tracing::trace!()`**. Simply importing `tracing::debug` has actually broken the original semantics.

Reply:
Thanks for the hint!